ja/support/index.html

```diff
@@ -2,148 +2,22 @@
-</section>
+  <p>以下の企業は、Groongaのサポートサービスを提供しています（五十音順）</p>
 
-<section id="menu">
-  <h2>サービスメニュー</h2>
-  <table>
-    <thead>
-      <tr>
-	<th>フェーズ</th>
-	<th>検討・設計</th>
-	<th>開発・構築</th>
-	<th>運用</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-	<th rowspan="4">サービス内容</th>
-	<th>Groonga導入支援</th>
-	<th>Groonga開発支援</th>
-	<th>Groongaサポート</th>
-      </tr>
-      <tr>
-	<td>
-	  <ul>
-	    <li>システム構成の検討</li>
-	    <li>性能検証、機能検証</li>
-	    <li>他の検索エンジンからの移行</li>
-	    <li>テスト導入</li>
-	  </ul>
-	</td>
-	<td>
-	  <ul>
-	    <li>インストール</li>
-	    <li>コーディング</li>
-	    <li>テスト開発</li>
-	  </ul>
-	</td>
-	<td rowspan="2">
-	  <ul>
-	    <li>
-	      問合せ対応
-	      <ul>
-		<li>インストール・アップデート方法の紹介</li>
-		<li>ニーズにあわせた機能の紹介</li>
-		<li>設定方法の案内</li>
-	      </ul>
-	    </li>
-	    <li>
-	      障害対応
-	      <ul>
-		<li>障害解析、ソースコード解析</li>
-		<li>回避策の提案、修正版の提供</li>
-	      </ul>
-	    </li>
-	    <li>
-	      その他
-	      <ul>
-		<li>パフォーマンス分析</li>
-		<li>システム拡張に関する相談への対応</li>
-		<li>Groonga他採用OSSに関する問合せへの対応</li>
-	      </ul>
-	    </li>
-	  </ul>
-	</td>
-      </tr>
-      <tr>
-	<td>
-	  <ul>
-	    <li>ミーティング参加</li>
-	    <li>検証環境構築支援、レポート作成</li>
-	    <li>システム設計の相談への対応</li>
-	    <li>Groonga他採用OSSに関する問合せへの対応</li>
-	  </ul>
-	</td>
-	<td>
-	  <ul>
-	    <li>開発、構築時の問合せ対応</li>
-	    <li>コードレビュー</li>
-	    <li>サンプルコード提供</li>
-	    <li>ユニットテスト開発</li>
-	    <li>不具合発見時の原因調査</li>
-	    <li>Groonga他採用OSSに関する問合せへの対応</li>
-	    <li>ミーティング参加</li>
-	  </ul>
-	</td>
-      </tr>
-      <tr>
-	<td colspan="4" class="note">
-	  Groongaに対する機能追加等のご要望には可能な範囲で対応します。
-	</td>
-      </tr>
-      <tr>
-	<th>費用</th>
-	<td>
-	  <ul>
-	    <li>ミーティング参加 40,000円/回から</li>
-	    <li>その他費用については作業内容についてヒアリングを実施した上で、お見積もりいたします。</li>
-	  </ul>
-	</td>
-	<td>
-	  <ul>
-	    <li>作業内容についてヒアリングを実施した上で、お見積もりいたします。</li>
-	  </ul>
-	</td>
-	<td>
-	  <ul>
-	    <li>インシデントサポート 50万円/年～</li>
-	    <li>サービスレベル、サポート内容に応じてお見積もりいたします。</li>
-	  </ul>
-	</td>
-      </tr>
-      <tr>
-	<th>サービス体制</th>
-	<td colspan="3">
-	  <p>サービスの提供元は有限会社未来検索ブラジルとなります。</p>
-	</td>
-      </tr>
-    </tbody>
-  </table>
-</section>
+  <h3>株式会社クリアコード</h3>
 
-<section id="contact">
-  <h2>問い合わせ先</h2>
-  <p>本件サービスに関するお問い合せ・見積依頼は、クリアコードにて承ります。</p>
-  <dl>
-    <dt>株式会社クリアコード</dt>
-    <dd>
-      <p>〒170-0005 東京都豊島区南大塚3丁目29番9号 中野ビル3階</p>
-    </dd>
-    <dt>問い合わせ方法<dt>
-    <dd>
-      <dl>
-        <dt>Web</dt>
-        <dd><a href="http://www.clear-code.com/contact/">http://www.clear-code.com/contact/</a></dd>
-        <dt>電話</dt>
-        <dd>03-5927-9440</dd>
-      </dl>
-    </dd>
-    <dt>担当</dt>
-    <dd>南、須藤</dd>
-  </dl>
+  <div class='company'>
+    <div class='logo'>
+      <a href='https://www.clear-code.com/'>
+        <img src='/images/support/ClearCode.svg'>
+      </a>
+    </div>
+    <div class='description'>
+      <p>クリアコードは2008年からGroongaの開発に参加しています。Groonga、PGroonga、Mroongaを採用したシステム開発、保守の支援を提供しています。</p>
```

Comment:
It might be a good idea to add links to PGroonga and Mroonga.

For example, how about doing as follow?

```suggestion
      <p>クリアコードは2008年からGroongaの開発に参加しています。Groonga、<a href="https://pgroonga.github.io/">PGroonga</a>、<a href="http://mroonga.org/ja/">Mroonga</a>を採用したシステム開発、保守の支援を提供しています。</p>
```